lib/app.js

```diff
@@ -9,23 +9,43 @@ function app (opts) {
       throw Error('Either id or appId is required');
     }
     const idField = opts.id ? 'id' : 'bundleId';
-    const idValue = opts.id || opts.appId;
-    resolve(common.lookup([idValue], idField, opts.country, opts.requestOptions));
+    if (typeof opts.id === 'string') opts.id = [opts.id];
```

Comment:
don't mutate the external opts object, use local const variables instead. Also the name `inputData` doesn't reflect what the field contains.